Formula/llvm.rb

```diff
@@ -281,7 +281,7 @@ def caveats
            "-std=c++11", "-stdlib=libc++", "test.cpp", "-o", "testlibc++",
            "-L#{opt_lib}", "-Wl,-rpath,#{opt_lib}"
     assert_includes (testpath/"testlibc++").dynamically_linked_libraries,
-                    opt_lib/shared_library("libc++", "1")
+                    (opt_lib/shared_library("libc++", "1")).to_path
```

Comment:
Bit confused by this, since `opt_lib/shared_library("libc++", "1")` should already be a `Pathname`. Do you need it to be a string? (Though I would've thought it would readily be coerced into one when needed.)

Reply:
Yeah, without the `.to_path` I get an error:

```
Minitest::Assertion: Expected ["/home/chandlerc/.linuxbrew/opt/llvm/lib/libc++.so.1", "/home/chandlerc/.linuxbrew/opt/llvm/lib/libunwind.so.1", "/home/chandlerc/.linuxbrew/opt/llvm/lib/libc++abi.so.1", "/lib/x86_64-linux-gnu/libm.so.6", "/lib/x86_64-linux-gnu/libgcc_s.so.1", "/lib/x86_64-linux-gnu/libc.so.6"] to include #<Pathname:/home/chandlerc/.linuxbrew/opt/llvm/lib/libc++.so.1>.
```

(Again, I'm a Ruby novice, so I was just hitting this with the hammer indicated by my testing...)

Reply:
I guess I could map the dynamically linked libraries array into an array of `Pathname`, but that seemed like more code.

Reply:
Ah. What you want there is `#to_s`:
```suggestion
                    (opt_lib/shared_library("libc++", "1")).to_s
```
which converts the `Pathname` to a string.

Reply:
Sure, the docs I read seemed to indicate that `to_path` was the canonical name and `to_s` an alias, but 🤷🏻